Gaijinizer patch/Map369.txt

```diff
@@ -268,667 +320,856 @@ You don't have to be so concerned...#MaybeWrong?
-
+> CONTEXT: Map369/events/5/pages/1/150/Dialogue/kazuya
+So, was the reason that you wanted me on your
+show because of how much you admired Nina?#Ron
 > END STRING
 
 > BEGIN STRING
 うーん…………
 
 その話題に入る前に、\\N[1]くんに一言
-> CONTEXT: Map369/events/5/pages/1/155/Dialogue/enemy24 < UNTRANSLATED
-
+> CONTEXT: Map369/events/5/pages/1/155/Dialogue/enemy24
+Hmmm, there's something I'd like to say to you
+before we start on that topic.#Ron
 > END STRING
 
 > BEGIN STRING
 \\{\\{"ちゃん"をつけろよ乳首野郎！！
-> CONTEXT: Map369/events/5/pages/1/159/Dialogue/enemy27 < UNTRANSLATED
-
+> CONTEXT: Map369/events/5/pages/1/159/Dialogue/enemy27
+\\{\\{Use "chan" after her name you
+damned nipple-crazed bastard!#Ron
 > END STRING
 
 > BEGIN STRING
 ………くらい言ってやりたいんだけど
 
 キミとニーナちゃんが『そういう関係』だってのは
 分かってるからあえて言わない。
-> CONTEXT: Map369/events/5/pages/1/161/Dialogue/enemy24 < UNTRANSLATED
+> CONTEXT: Map369/events/5/pages/1/161/Dialogue/enemy24
+Is what I'd like to say...
 
+But I know you and Nina-chan have "that kind" of
+relationship going on, so I won't say it.#Ron
 > END STRING
 
 > BEGIN STRING
 言ってるじゃん………
 
 乳首野郎ってオイ…
-> CONTEXT: Map369/events/5/pages/1/166/Dialogue/kazuya < UNTRANSLATED
+> CONTEXT: Map369/events/5/pages/1/166/Dialogue/kazuya
+Didn't you just say it?
 
+Also, "nipple-crazed bastard"?#Ron
 > END STRING
 
 > BEGIN STRING
 僕、こんな内気な性格でしょ？
 
 だからああやって、大舞台で明るく喋れるのが羨ましくって…
 いつか、ああいう司会をやってみたいって思ったんだ。
-> CONTEXT: Map369/events/5/pages/1/171/Dialogue/enemy26 < UNTRANSLATED
-
+> CONTEXT: Map369/events/5/pages/1/171/Dialogue/enemy26
+As you know, I've got a pretty bashful
+personality. I was jealous that she could go up
+on the big stage all bright and bubbly. I wanted
+to host a show of my own one day.#RonIffyWording
 > END STRING
 
 > BEGIN STRING
 でも…………今だから思うんだけど
 僕はきっと、『アイドル』になりたかったわけじゃない。
 
 『ニーナちゃん』になりたかったんだと思う……
-> CONTEXT: Map369/events/5/pages/1/176/Dialogue/enemy26 < UNTRANSLATED
+> CONTEXT: Map369/events/5/pages/1/176/Dialogue/enemy26
+But... thinking about it now...
 
+It's not that I wanted to become an idol like her,
+I think I actually just wanted to be her.#Ron
 > END STRING
 
 > BEGIN STRING
 カメラの向こうで、大勢が見てるって分かってるのに
 あんなに笑顔で、明るく振る舞って…『ホンモノ』を見せる
 
 まさしく、僕が求める『理想の姿』だったから…
-> CONTEXT: Map369/events/5/pages/1/181/Dialogue/enemy26 < UNTRANSLATED
-
+> CONTEXT: Map369/events/5/pages/1/181/Dialogue/enemy26
+Facing the cameras, knowing I'm being watched by
+huge numbers of people, yet still bright and with
+a huge smile on my face, showing them all the real
+me. That was the "ideal" I was chasing.#Ron
 > END STRING
 
 > BEGIN STRING
 だから、僕もニーナちゃんみたいに自信を付けたくて
 カメラを意識した、番組を進行するような感じで
 
 毎日毎日、バーゲストと一緒に特訓してたんだ！
-> CONTEXT: Map369/events/5/pages/1/186/Dialogue/enemy25 < UNTRANSLATED
-
+> CONTEXT: Map369/events/5/pages/1/186/Dialogue/enemy25
+That's why I had that Barghest help me train
+day after day to build the confidence to be
+conscious of the camera and properly progress
+a show like Nina-chan.#Ron
 > END STRING
```

Comment:
progress -> "host" or "run", and and replace the final period with an exclamation point.